File: CursoPOO/Python/uberBlack.py
from car import Car

class UberBlack(Car):
    typeCarAccepted = []
    seatsMasterial = []
    
    def __init__(self, license, driver, typeCarAccepted, seatsMasterial):
        super().__init__(license, driver)
        self.typeCarAccepted = typeCarAccepted 
        self.seatsMasterial = seatsMasterial

File: CursoPOO/Java/UberBlack.java
import java.util.ArrayList;
import java.util.Map;

class UberBlack extends Car {
    /*La palabra super hará referencia a los atributos y métodos de la super clase y this hará referencia a los atributos de la clase hija*/ 
    Map<String, Map<String,Integer>> typeCarAccepted;
    ArrayList<String> seatsMaterial;

    public UberBlack(Integer id, String license, Driver driver, Integer passegenger, Map<String, Map<String,Integer>> typeCarAccepted, ArrayList<String> seatsMaterial){
        super( license, driver);
        this.typeCarAccepted = typeCarAccepted;
        this.seatsMaterial = seatsMaterial;
    }
        
    
}
